packages/react-components/.eslintrc.js

```diff
@@ -46,7 +46,10 @@ module.exports = {
                 "@typescript-eslint/no-explicit-any": "off",
 
                 "jsx-control-statements/jsx-jcs-no-undef": "off",
-                "@typescript-eslint/ban-ts-comment": "off"
+                "@typescript-eslint/ban-ts-comment": "off",
+
+                "@typescript-eslint/member-delimiter-style": ["warn"]
```

Comment:
You should add it to https://github.com/gsoft-inc/sg-eslint/tree/master/packages/typescript instead